inst/traduire/fr-translation.json

```diff
@@ -114,8 +114,8 @@
     "STATUS_CALIBRATION_STRATA_LABEL": "État des strates d’étalonnage connues: ",
     "INFECTIONS_CALIBRATION_LEVEL_LABEL": "Nouveau niveau d’étalonnage des infections:  ",
     "INFECTIONS_CALIBRATION_STRATA_LABEL": "Nouvelles strates d’étalonnage des infections: ",
-    "PLHIV_CALIBRATION_LEVEL_LABEL": "Niveau d’étalonnage des PVVIH: ",
-    "PLHIV_CALIBRATION_STRATA_LABEL": "Strates d’étalonnage des PVVIH:  ",
+    "PLHIV_CALIBRATION_LEVEL_LABEL": "Niveau de calibration PVVIH:  ",
+    "PLHIV_CALIBRATION_STRATA_LABEL": "Catégorie d'étalonnage PVVIH: ",
     "POPULATION_CALIBRATION_LEVEL_LABEL": "Étalonnage de la population: ",
```

Comment:
@rtesra there is a mix of spaces and tabs after the `:` in these strings. Is that important?